apps/blockchain/lib/blockchain/block.ex

```diff
@@ -40,7 +40,9 @@ defmodule Blockchain.Block do
   end
 
   def compute_hash(%Block{index: i, previous_hash: h, timestamp: timestamp, data: data, nounce: nounce}) do
-    :crypto.hash(:sha256, "#{i}#{h}#{timestamp}#{Data.hash(data)}#{nounce}") |> Base.encode16
+    "#{i}#{h}#{timestamp}#{Data.hash(data)}#{nounce}"
+    |> Crypto.hash(:sha256)
+    |> Base.encode16
```

Comment:
I would add parenthesis in all functions call so you prevent any Elixir warning about it

Reply:
indeed I'm surprised I don't get any warnings from credo neither from elixir 🤔